client/termdb/TermTypeSearch.ts

```diff
@@ -13,7 +12,12 @@ The tree target is used to determine the allowed term types.
  */
 
 const useCases = {
-	matrix: [TermTypeGroups.DICTIONARY_VARIABLES, TermTypeGroups.MUTATION_CNV_FUSION],
```

Comment:
here worth to add a comment and state that this design is deliberate so when introducing a new term, it can be gradually added to each usecase and get enabled in the plot/target code. looks redundant but necessary

Reply:
I think we can start listing the cases excluded instead. Working on it.

Reply:
yes good idea